awx/ui/test/e2e/tests/test-xss.js

```diff
@@ -237,6 +238,7 @@ module.exports = {
         client.sendKeys('div.at-Panel smart-search input', `id:>${data.notification.id - 1} id:<${data.notification.id + 1}`);
         client.waitForElementNotPresent('div.at-Panel smart-search .SmartSearch-searchButton--disabled');
         client.waitForElementNotVisible('.overlay');
+        client.pause(2000);
```

Comment:
We definitely need to check what is blocking us to click and wait until it is gone.